app/common/public/locales/en/translation.json

```diff
@@ -281,5 +281,13 @@
   "invalidCapType": "Invalid capability type: {{type}}",
   "whitespaceDetected": "Text Starts and/or Ends With Whitespace",
   "pcloudyCredentialsRequired": "pCloudy username and api key are required!",
-  "duplicateCapabilityNameError": "A capability set with the same name already exists"
+  "duplicateCapabilityNameError": "A capability set with the same name already exists",
+  "errorLoadingGestures": "Error loading gestures",
+  "unableToUploadGestureFiles": "Could not upload gesture from below files:",
+  "gestureEmptyTickError": "Actions should have valid ticks.",
+  "gestureInvalidEventError": "Invalid event type '{{invalidEvent}}'. Valid values are {{validEvents}}.",
+  "gestureRequiredFieldsError": "Value '{{fields}}' are required for '{{eventType}}' event.",
+  "gestureNameCannotBeEmptyError": "Gesture name cannot be empty.",
+  "gestureInvalidJsonError": "Invalid JSON file. Unable to parse the file content.",
```

Comment:
It might be better to remove the dots at the end of these sentences, since most translation strings also don't have them